<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MyTcpClient</class>
 <widget class="QMainWindow" name="MyTcpClient">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1010</width>
    <height>759</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MyTcpClient</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="lblHost">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>30</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Host:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblPort">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>80</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Port:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblUsername">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>140</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Username:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblMessage">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>210</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Message</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="editMessage">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>250</y>
      <width>841</width>
      <height>191</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnStart">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>470</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Start Chat</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSend">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>470</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Send Message</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnClose">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>470</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Close Chat</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="editHost">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>20</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="editPort">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>80</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="editUsername">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>140</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lblOutput">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>540</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Output</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="editOutput">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>570</y>
      <width>841</width>
      <height>131</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1010</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
